### ISSUE TITLE
Poorly specified licensing for some lenses

### ISSUE THREAD

**Issue by berrange** [hercules-team/augeas#816]
I'm auditing the licensing of Augeas for the purposes of Fedora's SPDX license record. 

Most of the lenses just refer to being under the same license as the rest of Augeas, which is great. I came across a few lenses whose license is ill defined though:

* activemq_conf.aug
* activemq_xml.aug
* jettyrealm.aug
* jmxaccess.aug
* jmxpassword.aug

These all merely say "LGPL" without giving any version at all, so could be 2.0 or 2.1 with or without "or later". A charitable interpretation would be to assume the same terms as Augeas itself, which would be and thus LGPL 2.1 or later. It would be better if these were explicit like most other lens which say "This file is licenced under the LGPL v2+, like the rest of Augeas."

* dhcpd.aug
* ssh.aug

These all merely say "GPL" without giving any version at all, so could have been intended to be 1.0, 2.0, 3.0, with or without "or later". In absence of any data I would probably have to assume the oldest version 1.0, but really the original authors need to confirm this to make their intention explicit.  Possibly "GPL" was even an accident and it was simply meant to be LGPL 2.1 or later, like the rest of Augeas.

* mdadm_conf.aug

Just says "LGPL v2+", no distinction of 2.0 or 2.1, but it is probably fairly safe to assume the same as augeas itself as LGPL 2.1 or later

* syslog.aug

Just says "BSD". This one is a big problem, as there are soo many variants of BSD license available.

In summary, the three lenses saying "GPL" and "BSD"  are the significant problems here as there are so many possible variants in both cases that it is hard to assume intent. The original authors need contacting to clarify IMHO.

**Comment by igalic** [comment]
given that `syslog.aug` is authored by `mat@FreeBSD.org`, maybe we can just assume [FreeBSD's default BSD-2](https://docs.freebsd.org/en/articles/committers-guide/#pref-license). not sure if mat@ is on GitHub to ask him directly.

**Comment by mat813** [comment]
Yeah, bsd-2 feels right.
